elastic/kibana
# Add telemetry to risk inputs on entity flyout

## Summary

Add EBT events for risk summary inside the entity flyout.

* User collapses or expands the Risk Summary view
* User launches the expanded flyout to view risk inputs (Risk inputs or Expand flyout)
* User adds risk inputs to the timeline (These are implicitly tracked from the table, so please double check)



Sample of sent events:
```
"Toggle Risk Summary Clicked" 
event_type: "Toggle Risk Summary Clicked"
properties: {entity: 'host', action: 'hide'}

event_type: "Toggle Risk Summary Clicked"
properties: {entity: 'host', action: 'show'}

event_type: "Risk Inputs Expanded Flyout Opened"
properties: {entity: 'host'}

event_type: "Add Risk Input To Timeline Clicked"
properties: {quantity: 1}

event_type: "Add Risk Input To Timeline Clicked"
properties: {quantity: 5}

event_type: "Toggle Risk Summary Clicked"
properties: {entity: 'user', action: 'hide'}

event_type: "Toggle Risk Summary Clicked"
properties: {entity: 'user', action: 'show'}

event_type: "Risk Inputs Expanded Flyout Opened"
properties: {entity: 'user'}
```


### How to test it?
* Open kibana and generate alerts
* Enable the risk engine (make sure it generates data)
* Open the alerts page and click on host|user name
* Interact with the risk score component and open the expandable flyout
* The telemetry events should be logged on the console

### Dashboard
https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/r/s/Qt437
![Screenshot 2024-01-09 at 11 33 03](https://github.com/elastic/kibana/assets/1490444/5a903c3d-458c-479f-a6ee-6ce593107d3d)
